src/main/java/com/google/sps/COMMONS.java

```diff
@@ -19,9 +19,9 @@
 
 /** Definition of constants that servlets will use. */
 public interface COMMONS {
-  float MAX_DISTANCE_CLIENT = 20000f; // 20 kilometers
-  float MAX_DISTANCE_VENDOR = 3000f; // 3 kilometers
-  float MIN_DISTANCE = 0f;
+  int MAX_DISTANCE_CLIENT = 20000; // 20 kilometers
+  int MAX_DISTANCE_VENDOR = 3000; // 3 kilometers
+  int MIN_DISTANCE = 0;
```

Comment:
Hmm... I coded the geoHash using integers and seems logic, should we handle queries from `1.83m`, for example ?

Reply:
+1 on this. I think it would be better to use float to get the possibility to work on meters too (with this implementation  the minimum differential distance would be of 1km)

Reply:
I see.
We could handle those queries, just to have that option to be more precise, but with whole numbers is fine, after all we are working with meters, so decimals are like less than a meter.